nimbus/src/sampling.rs

```diff
@@ -143,7 +143,7 @@ fn is_hash_in_bucket(
 /// # Errors
 /// returns an error if the fraction not within the 0-1 range
 fn fraction_to_key(fraction: f64) -> Result<String> {
-    if fraction < 0.0 || fraction > 1.0 {
+    if !(0.0..=1.0).contains(&fraction) {
```

Comment:
heh - this is cute and all, but not sure the intent is as clear as the "obvious" version, but 🤷  :)

Reply:
TBH, I was trying to fix the clippy errors. This has nothing to do with the rest of this patch.